### PR TITLE
add progress bar component

### PR DESCRIPTION
**Pull request by EllaFan**
add progress bar component

### CODE REVIEW

File: src/common/dr-ng-widgets/js/progress.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+};
+
+function drProgressBar() {
+  return {
+    scope: {
+      current: '=current',
+      total: '=total'
+    },
+    restrict: 'EA',
+    template: '<div class="dr-ng-progress-container">'  +
```
Comment (liujingbreak) [inline src/common/dr-ng-widgets/js/progress.js]:
```
template: require('./views/progress-bar.html')
```
Move all '<div ...' stuff to a new  file "views/progress-bar.html"

File: src/common/dr-ng-widgets/styles/progress.less [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  }
+
+  .dr-ng-progress-num {
+    position: absolute;
+    right: 0;
+    top:0;
+    width: 40px;
+    height: 100%;
+    color: #999999;
```
Comment (liujingbreak) [inline src/common/dr-ng-widgets/styles/progress.less]:
```
@import "npm://@dr/dr-css-components/less/_colors.less"
```
用里面的LESS variable

File: src/common/dr-ng-widgets/styles/progress.less [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    font-size: 12px;
```
Comment (liujingbreak) [inline src/common/dr-ng-widgets/styles/progress.less]: use @import "npm://@dr/dr-css-components/less/_variables.less"

File: src/common/dr-ng-widgets/styles/progress.less [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  width: 100%;
+  height: 11px;
+  overflow: hidden;
+  .dr-ng-progress-bar {
+    margin:4px 40px 0 0;
+    height: 3px;
+    .total {
+      width: 100%;
+      height: 100%;
+      background-color: #f0f0f0;
```
Comment (liujingbreak) [inline src/common/dr-ng-widgets/styles/progress.less]:
```
@import "npm://@dr/dr-css-components/less/_colors.less"
```

**Comment by EllaFan** [comment]
@liujingbreak @claire1028 重新review下

**Pull request merged**